static/js/errorlogs/viewlogs.js

```diff
@@ -1,4 +1,20 @@
 MEDUSA.errorlogs.viewlogs = function() {
+    if (MEDUSA.config.fanartBackground) {
```

Comment:
Didn't see the common init file at the time and was editing page by page... I'll change it tonight.